README.md

```diff
@@ -41,9 +46,19 @@ Note: The version tested with Zivid cameras is 2.1.0.
 
 2. [**Download Zivid Sample Data**](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/450363393/Sample+Data).
 
-3. Launch Visual Studio 2017.
 
-4. Open and run one of the samples.
+Launch the Command Prompt by pressing *Win + R* keys on the keyboard, then type cmd and press Enter.
+
+Navigate to a location where you want to clone the repository, then run to following command:
+
+```bash
+git clone https://github.com/zivid/zivid-csharp-samples
+```
+
+Open *ZividNETSamples.sln* in Visual Studio, build it, and run it. If you are uncertain about doing this, check out our [**tutorials for configuring and building C# Samples**](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/534708242).
+
+Some of the samples depend on external libraries, in particular *YamlDotNet* and *MathNet.Numerics*. These libaries will be installed automatrically through NuGet when building the sample.
```

Comment:
You should spell check this file. I recommend enabling the spell checker in whichever editor you use.

Reply:
done